website/pages/contributing.mdx

```diff
@@ -60,13 +68,31 @@ reload the site.
 
 Project setup
 
-```sh
+<br/>
+
+<CodeGroup lang="sh">
+  <CodeTab label="Yarn">
+
+```
 yarn bootstrap # Installs all dependencies and links packages
 yarn dev # Runs dev build
```

Comment:
After thinking about it for a while, I no longer actually use the development build when working on components. (Because of storybook). Maybe it's better for contributors to actually run `yarn build` instead of `yarn dev` here.

What are your thoughts?

Reply:
I think I would second this, the builds are relatively the same minus a couple output formats in dev. In addition to that I'm pretty sure you cant run the local nuxt server for the docs site without the cjs bundle which is only output by a prod build. This was a note I made while getting started that I still want to include in the contributing page.

Reply:
Awesome! Let's do it!